src/app.scss

```diff
@@ -1,3 +1,3 @@
 .App {
```

Comment:
let's try to use lowercase. Conventions vary but generally title case is reserved for global css